db/migrate/20230206153448_add_get_an_identity_id_sync_bool_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddGetAnIdentityIdSyncBoolToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :get_an_identity_id_synced_to_ecf, :boolean, default: false
```

Comment:
Once the sync is completed and the data is flowing across properly this will be removed